price-estimator/swagger-ui.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
```

Comment:
Personally, I'm kind of against having `price-estimator` host the swagger UI HTML. Its fairly easy to go to https://petstore.swagger.io/ and just input `http://localhost:8080/openapi/openapi.yaml` to get the UI. I think just having the OpenAPI definition is enough.

If we really want to include host the swagger UI, I think it should be added in a separate PR.

Reply:
Ah so we would only host the one file but not the full ui. I'll have to check if this will work with CORS (setting allow origin header).
I didn't think of this possibility when I made the PR but I do like self hosting all the required files because it works without any external website. Why are you against this?

Reply:
> I'll have to check if this will work with CORS

Good point. We may need to enable CORS correctly.

> Why are you against this?

Not against self-hosting, I just don't think it should be the `price-estimator` that hosts this, and maybe this should be hosted in the dev portal instead of the service. Another concern of mine is that the zip bundled UI may contain Swagger branding that we maybe aren't allowed to distribute.

Another thing to consider is using `swagger-ui(-dist)?` NPM package and host that in a little HTML page of our own instead of downloading from a ZIP bundle.